frontend/public/components/app.jsx

```diff
@@ -180,7 +180,6 @@ class App extends React.PureComponent {
 
           <Route path="/k8s/cluster/clusterroles/:name/add-rule" exact component={EditRulePage} />
           <Route path="/k8s/cluster/clusterroles/:name/:rule/edit" exact component={EditRulePage} />
-          <Route path="/k8s/cluster/clusterroles/:name" component={props => <ResourceDetailsPage {...props} plural="clusterroles" />} />
```

Comment:
This is unnecessary. The generic rule below handles this (along with other cluster scoped resources) in exactly the same way:

```jsx
<Route path="/k8s/cluster/:plural/:name" component={ResourceDetailsPage} />
```

https://github.com/openshift/console/blob/master/frontend/public/components/app.jsx#L202

Reply:
Having said that, I can't explain why this fixes the service worker error, which I don't like